common/app/views/fragments/image.scala.html

```diff
@@ -10,6 +10,19 @@
     isImmersiveMainMedia: Boolean = false
 )
 
+@immersivePortraitSource(largestImage: model.ImageAsset, hidpi: Boolean = false) = {
+    <source media="@if(hidpi) {
+        (orientation: portrait) and (-webkit-min-device-pixel-ratio: 1.25), (orientation: portrait) and (min-resolution: 120dpi)
+    } else {
+        (orientation: portrait)
+    }"
+    sizes="@{100 * (largestImage.width.toDouble / largestImage.height.toDouble)}vh"
```

Comment:
> Should this be a whole number?

If it was a whole number this is what would happen: 

Say you have an image with a width of 500px and height of 300px. The calculation would then be:

`100 * (500 / 300)vh`

With a double you would get:
`166.66... vh`

With integers it would be:
`100vh` (100 * 1vh)

In some cases it makes sense to use integers, but in this case we need a double (or float?)